src/session_concurrency/payload_dataset_manager.h

```diff
@@ -56,8 +58,6 @@ class PayloadDatasetManager {
 
   PayloadsMapType CreateSessionIdToPayloadsMap() const;
 
```

Comment:
`GetSessionID()` is moved from private to public since it is now called in the place where the session ID input is recorded for the profile export JSON.